### PR TITLE
Testsuites polish: handling <pre> better in errors

### PR DESCRIPTION
**Pull request by jonbell-lot23**
**Old:**
image

**New:**
image

Our pre was acting up. Fixed.

### CODE REVIEW

File: src/ui/components/Library/Team/View/NewTestRuns/TestRunSpecDetails.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export function TestRunSpecDetails() {
   if (!spec) {
     return (
       <div className="flex h-full w-full items-center justify-center p-2">
-        <div className={styles.standardMessaging}>Select a test to see its details here</div>
+<div className={styles.standardMessaging}>Select a test to see its details here</div>
```
Comment (callingmedic911) [inline src/ui/components/Library/Team/View/NewTestRuns/TestRunSpecDetails.tsx:28]: The indent seems not right. Can you run prettier on it?

**Review by callingmedic911 — APPROVED** [review]
Formatting change request, looks good otherwise.

File: src/ui/components/Library/Team/View/NewTestRuns/TestRunSpecDetails.tsx (hunk 1 above)
Comment (jonbell-lot23) [reply src/ui/components/Library/Team/View/NewTestRuns/TestRunSpecDetails.tsx]: Thanks! Looks like it was a GitHub merge thing, should be good now.

**Comment by jonbell-lot23** [comment]
The prettier issue persisted, so I fixed it over here: https://github.com/replayio/devtools/pull/9980 cc @callingmedic911

**Pull request merged**